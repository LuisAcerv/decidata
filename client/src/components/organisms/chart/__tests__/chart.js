import React from "react";
import { shallow } from "enzyme";
import Chart from "../";

describe("Chart", () => {
  it("should render correctly", () => {
    let wrapper;
    wrapper = shallow(<Chart />);
    expect(wrapper).toMatchSnapshot();
  });
});
