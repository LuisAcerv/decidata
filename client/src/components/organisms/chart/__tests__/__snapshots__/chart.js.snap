// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart should render correctly 1`] = `ShallowWrapper {}`;
